src/components/date-time-picker/date-time-picker.component.ts

```diff
@@ -240,4 +238,38 @@ export class DateTimePickerComponent implements AfterViewInit, OnDestroy {
             this.datepicker.setDateToNow();
         }
     }
+
+    _onTimezoneChange(timezone: DateTimePickerTimezone): void {
+        if (!timezoneComparator(this.datepicker.timezone$.value, timezone)) {
+            this.timezoneChange.emit(timezone);
+        }
+    }
+
+    /**
+     * Update the service with the new timezone value, falling back on the default if it is undefined or
+     * not present in `timezones`.
+     */
+    private setTimezone(timezone: DateTimePickerTimezone | undefined): void {
+        // if the user does not provide a timezone, set it to the current timezone and emit the change
+        if (!timezone) {
+            this.datepicker.timezone$.next(this.datepicker.getDefaultTimezone());
+            this.timezoneChange.emit(this.datepicker.timezone$.value);
+            return;
+        }
+
+        // Check if the timezone is available in the timezones list; if not, get the default timezone
+        if (this.datepicker.isTimezoneAvailable(timezone)) {
+            this.datepicker.timezone$.next(timezone);
+        } else {
+            // This is probably an unintended state so emit a warning
+            console.warn(`ux-date-time-picker: specified timezone ${JSON.stringify(timezone)} is not present in the timezones array.`);
```

Comment:
If I set an initial timezone that doesn't exist, this doesn't seem to revert the the default timezone:

![image](https://user-images.githubusercontent.com/20795331/75975439-6b9ed000-5ed0-11ea-9c4e-35a8244c8749.png)

Reply:
Fix added